### PR TITLE
Navigationcontroller

### PR DESCRIPTION
**Pull request by Ilhembsk**
I start the navigation i didn't finished because i have to do the view journey. I will finish after the view.

### CODE REVIEW

File: app/controllers/navigations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class NavigationsController < ApplicationController
+
+  before_action :set_navigation, only: [:show, :new, :update]
```
Comment (skhohar) [inline app/controllers/navigations_controller.rb:3]: no update method in navigation

File: app/controllers/navigations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  def visited
+    @navigation = Navigation.find(params[:place_id])
+  end
+
+  def navigation_decision
+  end
+
+  def update
```
Comment (skhohar) [inline app/controllers/navigations_controller.rb:23]: no use for an update method

File: app/controllers/navigations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  private
+
+  def set_navigation
+    @navigation = Navigation.find(params[:place_id])
+  end
+
+  def navigation_params
+    params.require(:navigation).permit(:user_id, :place_id, :starting_coordinate, :ending_coordinate, :dine, :time_deadline, :date)
```
Comment (skhohar) [inline app/controllers/navigations_controller.rb:37]: done not dine

**Review by skhohar — CHANGES_REQUESTED** [review]

File: app/controllers/navigations_controller.rb (hunk 1 above)
Comment (Ilhembsk) [reply app/controllers/navigations_controller.rb]: I remove update method.

File: app/controllers/navigations_controller.rb (hunk 3 above)
Comment (Ilhembsk) [reply app/controllers/navigations_controller.rb]: Done !

File: app/controllers/navigations_controller.rb (hunk 2 above)
Comment (Ilhembsk) [reply app/controllers/navigations_controller.rb]: I remove it.

**Pull request merged**